apps/test/unit/templates/sectionsRefresh/SectionsSetUpContainerTest.jsx

```diff
@@ -178,4 +178,33 @@ describe('SectionsSetUpContainer', () => {
 
     sinon.restore();
   });
+
+  it('passes url attribute to make a new section if save and create new is clicked', () => {
+    sinon
+      .stub(document, 'querySelector')
+      .withArgs('#sections-set-up-container')
+      .returns({
+        checkValidity: () => true,
+      })
+      .withArgs('meta[name="csrf-token"]')
+      .returns({
+        attributes: {content: {value: null}},
+      });
+    sinon
+      .stub(utils, 'queryParams')
+      .withArgs('showSectionCreationDialog')
+      .returns('true');
+    const fetchSpy = sinon.spy(window, 'fetch');
+
+    const wrapper = shallow(<SectionsSetUpContainer />);
+
+    wrapper
+      .find('Button')
+      .at(1)
+      .simulate('click', {preventDefault: () => {}});
+
+    expect(fetchSpy).to.have.been.called.once;
+
+    sinon.restore();
```

Comment:
I can't remember -- does this also restore the fetch spy?

Reply:
Following up on a conversation we had offline: We've concluded that sinon.restore() covers cleanup for all functions that are mocking existing functions. This covers almost all cases in this file, with the exception of reportSpy. I added a restore() at the end of that unit test for cleanliness.